src/popup/router/components/CheckBox.vue

```diff
@@ -10,6 +15,8 @@ export default {
   data() {
     return {
       checkedProxy: false,
+      checkbox_unchecked: browser.runtime.getURL('../icons/checkbox-unchecked.svg'),
```

Comment:
```suggestion
      checkboxUnchecked: browser.runtime.getURL('../icons/checkbox-unchecked.svg'),
```
to confirm with our linter